src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -132,7 +133,10 @@ const AircraftConfigurationPage = () => {
                 <SelectGroup>
                     {weightUnitButtons.map((button) => (
                         <SelectItem
-                            onSelect={() => setWeightUnit(button.setting)}
+                            onSelect={() => {
+                                setWeightUnit(button.setting);
+                                new Notification().showNotification({ title: 'RELOAD AIRCRAFT', theme: 'GAMEPLAY', message: 'Reload the aircraft to apply settings.' });
```

Comment:
BTW, removing this warning (and the need for it when changing weight unit entirely in #5344).